antora.yml

```diff
@@ -214,6 +214,8 @@ asciidoc:
       - "openai_translation"
       - "ollama_chat"
       - "ollama_embeddings"
+      - "aws_bedrock_chat"
```

Comment:
@rockwotj. Are `aws_bedrock_chat` and `gcp_vertex_ai_chat` enterprise connectors?

Reply:
yes. Thanks!